feature_engine/selection/drop_psi_features.py

```diff
@@ -354,8 +345,12 @@ def fit(self, X: pd.DataFrame, y: pd.Series = None):
         # If required exclude variables that are not in the input dataframe
         self._confirm_variables(X)
 
-        # find numerical variables or check those entered are present in the dataframe
-        self.variables_ = find_or_check_numerical_variables(X, self.variables_)
+        # # find variables or check those entered are present in the dataframe
+        (
+            self.cat_variables_,
+            self.num_variables_,
```

Comment:
Is it necessary for these two variables to be a new instance attribute? How about keeping them as simple variables?

Reply:
I guess @dlaprins  is copying from what we do in another selector that works with both numerical and categorical.

We probably do not need to generate these new attributes. We could drop it. But if we do, then the user would not be able to easily find out which variables the transformer considered numerical and which ones categorical.

Simple controversial example: discrete variables encoded as object, will be considered categorical. Will the user be aware of this? 

I am not sure what is best here. I appreciate your input.

Reply:
I see now that saving categorical and numerical variables as instance attributes have been done in [_prediction/base_predictor.py](https://github.com/feature-engine/feature_engine/tree/main/feature_engine/_prediction#L12) indeed. It can make sense to save them as instance attributes for that purpose, but in that case I would comment that documentation for the new instance attributes is missing.

On the other hand, in the [information value selector](https://github.com/feature-engine/feature_engine/blob/main/feature_engine/selection/information_value.py#L217), those are not saved.

I personally think that both approaches can be correct.

Reply:
Let's not save them. @dlaprins could we update the code please?

Reply:
Will do